scvi/dataset/dataset.py

```diff
@@ -77,6 +77,8 @@ def collate_fn(self, batch):
             torch.LongTensor(self.labels[indexes])
 
     def subsample_genes(self, p_genes=1., subset_genes=None):
+        if p_genes > self.X.shape[1] or p_genes == 1.:
```

Comment:
Should it be `>=` instead of `>`? Why is `p_genes == 1` a special case?

Reply:
Is p_genes a proportion or a count?

Reply:
Since I called `subsample_genes` in all `__init__`, I don't want subsampling happens when `p_genes` is 1. 
`p_genes` can be either. If it's smaller than 1, then it's a proportion. If it's an integer greater than 1, then it's a count.

Reply:
`p_genes` should always be either a proportion or a count. It's confusing to use it as both--easy to misunderstand for anyone besides us. Will you find another way to do it?

Reply:
Yes. Will fix it now. 😊

Reply:
In sklearn `train_test_split` function, there is a `train_size` argument that can either be float or int. That's what motivated my choice. Maybe the terminology `gene_size` would have been better suited. In any case, it's rather true that we only think in terms of the number of genes (rather than the proportions compared to initial number).